records_mover/records/targets/table/target.py

```diff
@@ -81,22 +81,35 @@ def move_from_fileobjs_source(self,
 
     def can_move_from_fileobjs_source(self) -> bool:
         driver = self.db_driver(self.db_engine)
-        return driver.can_load_from_fileobjs()
+        loader = driver.loader_from_fileobj()
+        if loader is None:
+            return False
+        # TODO: why do we need can_load_from_fileobjs if we can just return None?
+        return loader.can_load_from_fileobjs()
 
     def can_load_direct(self, scheme: str) -> bool:
         driver = self.db_driver(self.db_engine)
-        return driver.best_scheme_to_load_from() == scheme
+        loader = driver.loader_from_records_directory()
+        # TODO: What can we do to make this more typesafe?
+        assert loader is not None
+        return loader.best_scheme_to_load_from() == scheme
 
     def known_supported_records_formats(self) -> List[BaseRecordsFormat]:
         driver = self.db_driver(self.db_engine)
-        return driver.known_supported_records_formats_for_load()
+        loader = driver.loader()
+        # TODO: What can we do to make this more typesafe?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker